Ntuplizer/test/ntuplize_cfg.py

```diff
@@ -253,13 +269,14 @@
     extraFinalObjectBranches['e'].append(matchedGenLeptonBranches)
     extraFinalObjectBranches['m'].append(matchedGenLeptonBranches)
 
-if any(len(c) == 4 for c in channels):
-    from UWVV.Ntuplizer.templates.eventBranches import centralJetBranches
-    extraInitialStateBranches.append(centralJetBranches)
+#if any(len(c) == 4 for c in channels):
```

Comment:
Has this been moved elsewhere? I'm not immediately finding it.

Reply:
I think I never fixed this after I added the jet cleaning. These collections would need to be specially produced with from the cleaned jet producer. Is it really that useful? I never used it since you can get all of these pretty trivially from the main jet collection.

Reply:
As long as the appropriate branches end up in the ntuples one way or another, I don't care whether that happens here or somewhere else.

Reply:
As it stands there will be no centralJetBranches in the ntuples. I don't find this useful enough to add it, but others might.

Reply:
Ah, sorry, I misread and thought this was more than just the eta<2.4 stuff. Never mind, it's fine to get rid of it.

Reply:
I deleted everything to do with eta2p4 jets